core/src/main/java/org/frankframework/pipes/ReplacingInputStream.java

```diff
@@ -0,0 +1,171 @@
+/*
+   Copyright 2019-2024 WeAreFrank!
```

Comment:
Ik denk dat hier het oorspronkelijke copyright ook moet worden opgenomen, in ieder geval de naam & datum die ook hieronder staan.

Reply:
Please do still update the copyright notice here!